postman/schemas/paths/component/data-science-result.yaml

```diff
@@ -0,0 +1,28 @@
+# /component/data-science/{id}/result?runId=runId
+get:
+    summary: Get results about a specified run from the data-science component.
+    operationId: getDataScienceResult
+    parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - in: query
+          name: runId
+          required: true
+          schema:
+              type: string
+              format: uuid
+    responses:
+        "200":
+            description: Successfull operation
```

Comment:
I think successful is spelled with one `l`. Might need correction everywhere...

Reply:
Yes 😇 Thank you very much 🥳